src/components/Form.js

```diff
@@ -30,27 +32,28 @@ const Form = () => {
   };
 
   return (
-    <div>
-      <input
-        type="text"
-        placeholder="Title"
-        value={title}
-        onChange={(e) => setTitle(e.target.value)}
-      />
-      <br />
-      <br />
-      <input
-        type="text"
-        placeholder="Author"
-        value={author}
-        onChange={(e) => setAuthor(e.target.value)}
-      />
-      <br />
-      <br />
-      <button type="button" onClick={handleAddBook}>
-        Add Book
-      </button>
-      {message && <p>{message}</p>}
+    <div className="form-container">
+      <h3 className="form-title">ADD NEW BOOK</h3>
+      <div className="input-form">
+        <input
+          type="text"
+          placeholder="Book Title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+          className="book-input"
+        />
+        <input
+          type="text"
+          placeholder="Book Author"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+          className="author-input"
+        />
+        <button type="button" onClick={handleAddBook} className="add-btn">
+          Add Book
+        </button>
+      </div>
+      {message && <p className="error">{message}</p>}
```

Comment:
- [ ] As the previous reviewer suggested that whenever a user tries to add a book, the `Loading...` appears but the book doesn't get added and it doesn't show any books from API. I have cloned this project on my machine I ran the server on the latest commit. 
One thing I noticed is that there is an error when you run the server and another error when you try to add a book. Kindly consider fixing these errors.
your project 

https://user-images.githubusercontent.com/96683843/231849831-13f04496-f5b1-453c-a40c-91ade7935e69.mov


errors in console
![re](https://user-images.githubusercontent.com/96683843/231849864-b5cada32-adac-4ab1-9a1f-1ef95f414534.jpg)

Reply:
@Zuhaib042 Thank you very much for the feedback. I think I know the reason. This is based on the suggestion made by the yesterday's reviewer. I just hid my key and used .env instead. I will fix that and make it available.